docs/embeddable-uis/element/access-requests.mdx

```diff
@@ -0,0 +1,589 @@
+---
+sidebar_position: 2
+title: Access Requests API Documentation
+---
+import HelpSupportTile from "@site/src/components/elements/HelpSupportTile.js";
+import ActionContainer from "@site/src/components/elements/ActionContainer.js";
+import ActionTile from "@site/src/components/elements/ActionTile.js";
+import CodeDropdown from "@site/src/components/elements/CodeDropdown.js";
+import FlexWrapper from "@site/src/components/elements/FlexWrapper.js";
+import code from "@site/src/components/elements/codeBlock.json";
+
+# Access Requests API Documentation
+
+This documentation provides details about the Access Requests API.
```

Comment:
Which kind of details ? 
It gives you more details and examples of how to use Access Requests API within your app.

Reply:
fixed